.github/workflows/build-image.yml

```diff
@@ -0,0 +1,41 @@
+name: Build Image
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - uses: actions/checkout@v4
+
+      # TODO: Do templates even make sense now?
+      - name: Emit Templates
+        run: |
+          echo "hi"
+
+      # FIXME: We're missing env vars
+      - name: Build
+        id: build
+        uses: docker/bake-action@v6
+        with:
+          source: .
+          files: ./9.0/bake.hcl
```

Comment:
This should probably be a reusable workflow that takes the "family" as an input.